activesupport/test/core_ext/object/exclusion_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'abstract_unit'
+require 'active_support/core_ext/object/exclusion'
+
+class NotInTest < ActiveSupport::TestCase
+  def test_not_in_array
+    assert 1.not_in?([2, 3])
+    assert !2.not_in?([1,2])
```

Comment:
Only change the new tests added. We don't retroactively change styles, but we use the new style in new lines.